app/src/mocks/mock-blocks-table.tsx

```diff
@@ -0,0 +1,334 @@
+      era_end: null,
+      timestamp: '2023-06-23T18:14:15+00:00',
+      era_id: 3,
+      height: 3,
+      protocol_version: 'testProtocol',
+    },
+    body: { proposer: '22222222222', deploy_hashes: [], transfer_hashes: [] },
+    proofs: [],
+  },
+  {
+    hash: '11111111111',
+    header: {
+      parent_hash: '111',
+      state_root_hash: '222',
+      body_hash: '333',
+      random_bit: false,
+      accumulated_seed: 'testSeed',
+      era_end: null,
+      timestamp: '2023-06-23T18:14:15+00:00',
+      era_id: 4,
+      height: 4,
+      protocol_version: 'testProtocol',
+    },
+    body: { proposer: '22222222222', deploy_hashes: [], transfer_hashes: [] },
+    proofs: [],
+  },
+  {
+    hash: '11111111111',
+    header: {
+      parent_hash: '111',
+      state_root_hash: '222',
+      body_hash: '333',
+      random_bit: false,
+      accumulated_seed: 'testSeed',
+      era_end: null,
+      timestamp: '2023-06-23T18:14:15+00:00',
+      era_id: 5,
+      height: 5,
+      protocol_version: 'testProtocol',
+    },
+    body: { proposer: '22222222222', deploy_hashes: [], transfer_hashes: [] },
+    proofs: [],
+  },
+  {
+    hash: '11111111111',
+    header: {
+      parent_hash: '111',
+      state_root_hash: '222',
+      body_hash: '333',
+      random_bit: false,
+      accumulated_seed: 'testSeed',
+      era_end: null,
+      timestamp: '2023-06-23T18:14:15+00:00',
+      era_id: 6,
+      height: 6,
+      protocol_version: 'testProtocol',
+    },
+    body: { proposer: '22222222222', deploy_hashes: [], transfer_hashes: [] },
+    proofs: [],
+  },
+  {
+    hash: '11111111111',
+    header: {
+      parent_hash: '111',
+      state_root_hash: '222',
+      body_hash: '333',
+      random_bit: false,
+      accumulated_seed: 'testSeed',
+      era_end: null,
+      timestamp: '2023-06-23T18:14:15+00:00',
+      era_id: 7,
+      height: 7,
+      protocol_version: 'testProtocol',
+    },
+    body: { proposer: '22222222222', deploy_hashes: [], transfer_hashes: [] },
+    proofs: [],
+  },
+  {
+    hash: '11111111111',
+    header: {
+      parent_hash: '111',
+      state_root_hash: '222',
+      body_hash: '333',
+      random_bit: false,
+      accumulated_seed: 'testSeed',
+      era_end: null,
+      timestamp: '2023-06-23T18:14:15+00:00',
+      era_id: 8,
+      height: 8,
+      protocol_version: 'testProtocol',
+    },
+    body: { proposer: '22222222222', deploy_hashes: [], transfer_hashes: [] },
+    proofs: [],
+  },
+  {
+    hash: '11111111111',
+    header: {
+      parent_hash: '111',
+      state_root_hash: '222',
+      body_hash: '333',
+      random_bit: false,
+      accumulated_seed: 'testSeed',
+      era_end: null,
+      timestamp: '2023-06-23T18:14:15+00:00',
+      era_id: 9,
+      height: 9,
+      protocol_version: 'testProtocol',
+    },
+    body: { proposer: '22222222222', deploy_hashes: [], transfer_hashes: [] },
+    proofs: [],
+  },
+  {
+    hash: '11111111111',
+    header: {
```

Comment:
If the only property that's being changed between these 10 `blocks`, then we should probably generate this array dynamically in code.